src/GridFieldNestedForm.php

```diff
@@ -0,0 +1,517 @@
+    }
+
+    public function getColumnsHandled($gridField)
+    {
+        return ['ToggleNested'];
+    }
+
+    public function getColumnAttributes($gridField, $record, $columnName)
+    {
+        return ['class' => 'col-listChildrenLink grid-field__col-compact'];
+    }
+
+    public function augmentColumns($gridField, &$columns)
+    {
+        if (!in_array('ToggleNested', $columns)) {
+            array_splice($columns, 0, 0, 'ToggleNested');
+        }
+    }
+
+    public function getColumnContent($gridField, $record, $columnName)
+    {
+        if ($this->atMaxNestingLevel($gridField)) {
+            return '';
+        }
+        $gridField->addExtraClass('has-nested');
+        if ($record->ID && $record->exists()) {
+            $this->gridField = $gridField;
+            $relationName = $this->getRelationName();
+            if (!$record->hasMethod($relationName)) {
+                return '';
+            }
+            if ($this->canExpandCheck) {
+                if (is_callable($this->canExpandCheck)
+                    && !call_user_func($this->canExpandCheck, $record)
+                ) {
+                    return '';
+                } elseif (is_string($this->canExpandCheck)
+                    && $record->hasMethod($this->canExpandCheck)
+                    && !$record->{$this->canExpandCheck}($record)
+                ) {
+                    return '';
+                }
+            }
+            $toggle = 'closed';
+            $className = str_replace('\\', '-', get_class($record));
+            $state = $gridField->State->GridFieldNestedForm;
+            $stateRelation = $className.'-'.$record->ID.'-'.$this->relationName;
+            $openState = $state && (int)$state->getData($stateRelation) === 1;
+            $forceExpand = $this->expandNested && $record->$relationName()->count() > 0;
+            if (!$this->forceCloseNested
+                && ($forceExpand || $openState)
+            ) {
+                $toggle = 'open';
+            }
+
+            GridFieldExtensions::include_requirements();
+
+            return ViewableData::create()->customise([
+                'Toggle' => $toggle,
+                'Link' => $this->Link($record->ID),
+                'ToggleLink' => $this->ToggleLink($record->ID),
+                'PjaxFragment' => $stateRelation,
+                'NestedField' => ($toggle == 'open') ? $this->handleNestedItem($gridField, null, $record): ' '
+            ])->renderWith('Symbiote\GridFieldExtensions\GridFieldNestedForm');
+        }
+    }
+    
+    public function getURLHandlers($gridField)
+    {
+        return [
+            'nested/$RecordID/$NestedAction' => 'handleNestedItem',
+            'toggle/$RecordID' => 'toggleNestedItem',
+            'POST movetoparent' => 'handleMoveToParent'
+        ];
+    }
+
+    /**
+     * @param GridField $gridField
+     * @return array
+     */
+    public function getHTMLFragments($gridField)
+    {
+        /**
```

Comment:
```suggestion
        /*
```
The double asterisk tells IDEs that this comment may include typehints etc - but this is just a multi-line comment so it shouldn't have that.
I'm nitpicking here - I won't hold back merging if this change isn't made.